pkg/beholder/message.go

```diff
@@ -16,6 +18,8 @@ type Message struct {
 type Metadata struct {
 	//	REQUIRED FIELDS
 	// Schema Registry URI to fetch schema
+	BeholderDomain     string `validate:"required,domain_entity"`
+	BeholderEntity     string `validate:"required,domain_entity"`
```

Comment:
What is the significance of these being `required`? What happens when they are omitted?

Reply:
`required` is a built-in validator that ensures a field is non-empty. if it was omitted here then empty strings would be valid

Reply:
I mean that if these fields are not set, what happens at run time? (because I don't see them being set in this PR - have you tried importing this in to core yet?)

Reply:
Oh, I guess we are not actually using the `Message` type yet? :shrug: 
I'm still curious what happens here

Reply:
I have not tried importing this into core yet

at runtime, if the fields are not set, custom messages will fail to send when Emit is called ([see here](https://github.com/smartcontractkit/chainlink-common/blob/4b45ad16ad7fa0b865decac91f60aa77b91bf147/pkg/beholder/client.go#L242))

this is following the existing pattern we have for our other required field `BeholderDataSchema`, the next line in this file